frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Add_HPA_Action.feature

```diff
@@ -0,0 +1,57 @@
+Feature: Add HPA action and topology sidebar modifications
+	As a user, I want to add a HPA to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment/deployment-config 
+    And user has assigned values to cpu and memory value in resource limit section of advanced option
+
+@regression
+Scenario: Add HPA form
+   Given user is in topology
+   When user opens sidebar of workload
+   And user opens action menu
+   And user clicks on Add Horizontal Pod Autoscaler option 
+   Then user sees Add Horizontal Pod Autoscaler page opens with form view selected
+   And user sees name field 
+   And user sees minimum and maximum pods with increasing and decreasing button
+   And user sees CPU Utilization with % at the end
+   And user sees Memory Utilization with % at the end
+   And user sees save and cancel button at the end
+
+@regression
+Scenario: Add HPA to deployment/deployment-config
+   Given user is in topology
+   When user opens sidebar of workload
+   And user opens action menu
+   And user clicks on Add Horizontal Pod Autoscaler option 
+   And user sees Add Horizontal Pod Autoscaler page opens with form view selected
+   And user adds name 'example' 
+   And user increase minimum and maximum pods value to 2 and 5 respectively
+   And user gives CPU Utilization and Memory Utilization value as 60 and 30
+   And user clicks on save
+   And user is taken to topology
+   And user opens the sidebar
+   Then user can see two pods under pod section
+   And user can see Horizontal Pod Autoscalers section
+   And user sees the example with HPA tag associated present under HPA section
+
+@regression, @manual
```

Comment:
Can you make it from Context Menu so that we can cover all the scenarios?

Reply:
I didn't include that as it wasn't there in the cluster I tested

Reply:
Okay.